.circleci/config.yml

```diff
@@ -1,54 +1,61 @@
 version: 2
 jobs:
-  build:
+  build-project:
     docker:
-      - image: checkstyle/sonarqube-maven-git:7.7-community-8c1737e
+      - image: cimg/openjdk:11.0.16
 
+    steps:
+      - checkout
+      - run: mvn package -Pno-validations
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
+
+  execute-sonar:
+    docker:
+      - image: sonarqube:9.0-community
     working_directory: ~/repo
 
     environment:
       # Customize the JVM maximum heap limit
       MAVEN_OPTS: -Xmx3200m
 
     steps:
-      - checkout
+      - attach_workspace:
+          at: /root
+      - run:
+          name: Install packages
+          command: apk add maven
```

Comment:
Installation of maven takes less than 5 seconds, I do not think we need sonar image with maven pre-installed